src/System.Data.SqlClient/tests/System.Data.SqlClient.Tests.builds

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="FunctionalTests\System.Data.SqlClient.Tests.csproj" />
+    <Project Include="ManualTests\System.Data.SqlClient.ManualTesting.Tests.csproj" />
```

Comment:
So this is a work-around for the issue with run-tests where run-tests will fail if a csproj exists but no test binary and without an UnsupportedPlatform tag. Once run-tests is fixed, all the Unsupported tags can go away